cms/wp/db/backup/import.php

```diff
@@ -0,0 +1,20 @@
+<?php
+/**
+ * Wordpress Database Backup Import.
+ *
+ * @file
+ * @package wp-deployer
+ */
+
+namespace Deployer;
+
+task(
+	'wp:db:backup:import',
+	function () {
+		$latestBackup = within( '{{backups}}', "find tmp -type f -printf '%T@ %p\n' | sort -n | tail -1 | cut -f2- -d\" \"" );
```

Comment:
Should `tmp` be `.`?

Reply:
Yes, this was/is cary over from a little trick we used on a project that actually stored the db dump in the single "tmp" file, which eliminated file ambiguity but doesn't scale per say.  Will change to .

Reply:
@colinodell Actually, we should talk to @dl-unleashed-technologies because he uses the "tmp" specifically in the Drupal database scripts 

- https://github.com/unleashedtech/deployer-recipes/blob/main/cms/drupal/db/backup/import.php#L11
- https://github.com/unleashedtech/deployer-recipes/blob/main/cms/drupal/db/backup/download.php#L5

My preference is to NOT rename the files that are created on the server ( https://github.com/unleashedtech/deployer-recipes/blob/main/cms/drupal/db/backup/create.php#L23 ) but to make the import script more intelligent so it knows which (latest) db dump to import rather than a specific file name of "tmp".   

Let me know what you think.

Reply:
Indeed I don't like the tmp method. Currently it leaves the "tmp" file in the repo root.